### ISSUE TITLE
apostrophe should not be considered part of a word

### ISSUE THREAD

**Issue by Dominique-M** [sgsinclair/Voyant#367]
Cirrus considers "the history" as two words, but "l’histoire" as one. This is a serious limitation on the use of Cirrus on French corpus. If "histoire" appears in Cirrus, I don’t want to see "l’histoire" and I want the total frequency of "histoire", not "l’histoire" deduced.
In in English, "don't" is a default stop word and I could add "don’t". But elision of the article is very common in French and it would be impractical to include as stop words a whole dictionary of words beginning with a vowel combined with article and apostrophe. 
I suspect that this is not a simple problem. If "don’t" becomes to words, it would give a confusing frequency of "don".

**Comment by sgsinclair** [comment]
Yes indeed, this is a vexing issue. One of Voyant's strengths is that it handles a lot of languages adequately, though one of irs weaknesses is that it doesn't handle any one language particularly well. Even apostrophes in French are lexicales specific of course, you wouldn't normally want two words from "aujourd'hui" (though it may be possible to handle elided determiners specially).

One possible solution for now is to use to the wordBoundaries value of the tokenization option during corpus creation: http://voyant-tools.org/docs/#!/guide/corpuscreator-section-tokenization

Compare these two:

* http://voyant-tools.org/?input=l'histoire
* http://voyant-tools.org/?input=l'histoire&tokenization=wordBoundaries
